packages/containerd/containerd-config-toml_vmware-k8s

```diff
@@ -0,0 +1,31 @@
+version = 2
```

Comment:
I tried this with `vmware-dev` previously and couldn't get the build to work.  Same here:

```
$ ls -lah packages/containerd/
...
lrwxrwxrwx.  1 mrowicki mrowicki   30 Apr 23 16:03 containerd-config-toml_vmware-k8s -> containerd-config-toml_aws-k8s


$ cargo make -e BUILDSYS_VARIANT=vmware-k8s-1.19
...
  #17 32.99 install: cannot stat '/home/builder/rpmbuild/SOURCES/containerd-config-toml_vmware-k8s': No such file or directory
...
```

Reply:
That's unfortunate, but don't give up!  Instead of symlinks, we can rename the files to not be so specific.  We only have two configs, one for k8s and one for everything else - dev and ECS variants.  What if we call them `containerd-config-toml_k8s` and `containerd-config-toml_basic` or something similar?

Reply:
I think we should make this change in a separate PR as I'd rather not block this work on something that affects other variants. :)

Reply:
What if I do it as a separate PR merged before this one?

Reply:
https://github.com/bottlerocket-os/bottlerocket/pull/1538

Reply:
#1538 is merged, hopefully this is easily fixed now, just using the `containerd-config-toml_k8s` file :)